src/sample_submission_data.R

```diff
@@ -0,0 +1,9 @@
+## Script that generates sample submission data to pass validation checks
+require(arrow) # for parquet writing
```

Comment:
here::i_am("src/sample_submission_data.R")
require(arrow) # for parquet writing